app/src/components/points-of-interest/IAPP/IAPP-Site.tsx

```diff
@@ -24,46 +26,64 @@ const useStyles = makeStyles((theme) => ({
     width: "auto",
     tableLayout: 'auto'
   },
+  tableContainer: {
+    display: 'table-row'
+  },
   cell: {
-    whiteSpace: 'nowrap'
+    whiteSpace: 'nowrap',
+    width: '1px'
   },
   wideCell: {
-    whiteSpace: 'nowrap'
+    whiteSpace: 'normal',
+    minWidth: '500px'
   },
   missingValue: {
     fontStyle: 'italic',
     color: '#777'
+  },
+  header: {
+    backgroundColor: 'rgba(0, 0, 0, 0.06)'  // TODO pull from theme
+  },
+  dropdown: {
+    paddingBottom: 0,
+    paddingTop: 0,
+    paddingLeft: '1em'
+  },
+  dropdownCol: {
+    maxWidth: '2em'
   }
 }));
 
 export interface IAPPSitePropType {
   record: any;
 }
+
+
 export const IAPPSite: React.FC<IAPPSitePropType> = (props) => {
   const classes = useStyles();
+  const ifApplicable = (value) => (value && String(value).trim() != '') ? value : <div className={classes.missingValue}>N/A</div>;
```

Comment:
can we just do this?
```suggestion
  const ifApplicable = (value) => (value && String(value).trim()) ? value : <div className={classes.missingValue}>N/A</div>;
```